### PR TITLE
Port almost all modules to v10

### PR DESCRIPTION
**Pull request by alexis-via**

### CODE REVIEW

**Review by adrienpeiffer — APPROVED** [review]
Code review and some functional test on payment order, payment mode, sepa file generation.

File: account_banking_pain_base/migrations/8.0.0.2/post-migration.py [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # -*- coding: utf-8 -*-
```
Comment (pedrobaeza) [inline account_banking_pain_base/migrations/8.0.0.2/post-migration.py]: Remove this file. It's not needed in this branch.

File: account_banking_sepa_direct_debit/__manifest__.py [hunk 2]
```diff
@@ -7,7 +7,7 @@
 {
     'name': 'Account Banking SEPA Direct Debit',
     'summary': 'Create SEPA files for Direct Debit',
-    'version': '9.0.1.0.0',
+    'version': '10.0.1.0.0',
     'license': 'AGPL-3',
     'author': "Akretion, "
               "Serv. Tecnol. Avanzados - Pedro M. Baeza, "
```
Comment (pedrobaeza) [inline account_banking_sepa_direct_debit/__manifest__.py]: Can you please change all my author references to `Tecnativa`?

File: account_banking_sepa_credit_transfer/migrations/8.0.0.3/post-migration.py [hunk 3]
```diff
@@ -1,25 +1,9 @@
...
-#    You should have received a copy of the GNU Affero General Public License
-#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
+# © 2016 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
-from openerp import pooler, SUPERUSER_ID
+
+from odoo import pooler, SUPERUSER_ID
```
Comment (pedrobaeza) [inline account_banking_sepa_credit_transfer/migrations/8.0.0.3/post-migration.py]: Remove this file

File: account_banking_pain_base/post_install.py [hunk 4]
```diff
@@ -3,13 +3,13 @@
...
-    companies = pool['res.company'].browse(cr, SUPERUSER_ID, company_ids)
-    for company in companies:
-        pool['res.company']._default_initiating_party(
-            cr, SUPERUSER_ID, company)
+def set_default_initiating_party(cr, registry):
+    with api.Environment.manage():
+        env = api.Environment(cr, SUPERUSER_ID, {})
+        companies = env['res.company'].search([])
+        for company in companies:
+            env['res.company']._default_initiating_party(company)
```
Comment (Garamotte) [inline account_banking_pain_base/post_install.py]: The `_default_initiating_party` method should be converted from `@api.model` to `@api.multi` without the _self recordset_ argument, to follow v10.0 standard changes.
Then, you will be able to directly call `companies._default_initiating_party()`.

File: account_banking_sepa_credit_transfer/post_install.py [hunk 5]
```diff
@@ -2,18 +2,18 @@
...
-    if sct_id:
-        ajo.write(cr, SUPERUSER_ID, journal_ids, {
-            'outbound_payment_method_ids': [(4, sct_id)],
-        })
+def update_bank_journals(cr, registry):
+    with api.Environment.manage():
+        env = api.Environment(cr, SUPERUSER_ID, {})
+        ajo = env['account.journal']
+        journals = ajo.search([('type', '=', 'bank')])
+        sct_id = env['ir.model.data'].xmlid_to_res_id(
```
Comment (Garamotte) [inline account_banking_sepa_credit_transfer/post_install.py]: Isn't it better to use `env.ref(xml_id)` ?

File: account_banking_sepa_direct_debit/post_install.py [hunk 6]
```diff
@@ -2,17 +2,18 @@
...
-    if sdd_id:
-        ajo.write(cr, SUPERUSER_ID, journal_ids, {
-            'inbound_payment_method_ids': [(4, sdd_id)],
-        })
+def update_bank_journals(cr, registry):
+    with api.Environment.manage():
+        env = api.Environment(cr, SUPERUSER_ID, {})
+        ajo = env['account.journal']
+        journals = ajo.search([('type', '=', 'bank')])
+        sdd_id = env['ir.model.data'].xmlid_to_res_id(
```
Comment (Garamotte) [inline account_banking_sepa_direct_debit/post_install.py]: Isn't it better to use `env.ref(xml_id)` ?

**Comment by pedrobaeza** [comment]
@alexis-via, can you attend last comments from @sylvain-garancher and I'll merge?

**Comment by alexis-via** [comment]
@pedrobaeza I took into account the comments of @sylvain-garancher

**Pull request merged**